pages/api/build-metrics.js

```diff
@@ -0,0 +1,72 @@
+import { CACHE_DURATION_SECONDS } from 'common-util/constants';
+import { autonolasGraphClient } from 'common-util/graphql/client';
+import { totalBuildersQuery } from 'common-util/graphql/queries';
+
+const fetchTotalBuilders = async () => {
+  try {
+    const result = await autonolasGraphClient.request(totalBuildersQuery);
+    const globals = result?.globals || [];
+    if (globals.length === 0) {
+      return null;
+    }
+    return Number(globals[0]?.totalBuilders || 0);
```

Comment:
tiny: since global.length check is already done above, so
```suggestion
    if (globals.length === 0) {
      return null;
    }
    return Number(globals[0].totalBuilders || 0);
```